data/shared/citizen/scripting/lua/scheduler.lua

```diff
@@ -373,6 +376,9 @@ AddEventHandler(('on%sResourceStart'):format(IsDuplicityVersion() and 'Server' o
 			end)
 		end
 	end
+		
+	-- Clear cache when resource start, so we can get the new exports for a reloaded module
```

Comment:
It'd be a good idea to also clear on a resource stop, or one may accidentally call an invalid ref in the C++ engine, whereas the original code would error out.